### ISSUE TITLE
The readme feels a bit lengthy. Shorten it?

### ISSUE THREAD

**Issue by Mariellavanerve** [course-dprep/Airbnb-new-years-eve#72]

**Comment by Mariellavanerve** [comment]
I feel like a lot of the method is already said in the motivation, can we look at this together and delete some stuff? Mostly the method, but I'm afraid of deleting something which might be important.

**Comment by fleurlemire** [comment]
Yes lets look into this together! i have the same @Mariellavanerve

**Comment by Mariellavanerve** [comment]
We did this.
